DataFormats/L1Scouting/interface/OrbitCollection.h

```diff
@@ -0,0 +1,186 @@
+#ifndef DataFormats_L1Scouting_OrbitCollection_h
+#define DataFormats_L1Scouting_OrbitCollection_h
+
+#include "DataFormats/Common/interface/FillView.h"
+#include "DataFormats/Common/interface/fillPtrVector.h"
+#include "DataFormats/Common/interface/setPtr.h"
+#include "DataFormats/Common/interface/traits.h"
+#include "FWCore/Utilities/interface/GCCPrerequisite.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include <cstdint>
+#include <vector>
+template <class T>
+class OrbitCollection {
```

Comment:
This class template needs
```cpp
  CMS_CLASS_VERSION(3)
```
somewhere in the public section, and the file needs to
```cpp
#include "DataFormats/Common/interface/CMS_CLASS_VERSION.h"
```